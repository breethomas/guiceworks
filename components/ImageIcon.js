// @flow
import React from 'react'

type Props = {
  alt: string,
}

export default (props: Props) => (
  <img
    alt={props.alt}
    width={32}
    height={32}
    {...props}
  />
)
