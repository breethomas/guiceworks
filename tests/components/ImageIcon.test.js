import React from 'react'
import renderer from 'react-test-renderer'
import ImageIcon from '../../components/ImageIcon'

describe('ImageIcon', () => {
  let component

  beforeEach(() => {
    component = <ImageIcon alt="alt text" />
  })

  it('shows the correct snapshot for the ImageIcon component tree', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
