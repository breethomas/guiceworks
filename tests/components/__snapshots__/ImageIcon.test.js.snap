// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageIcon shows the correct snapshot for the ImageIcon component tree 1`] = `
<img
  alt="alt text"
  height={32}
  width={32}
/>
`;
